java/clients/src/test/java/sleeper/clients/admin/InstancePropertyReportTest.java

```diff
@@ -70,29 +71,56 @@ public void shouldPrintInstancePropertyReportWhenChosen() {
                 .contains("# The S3 bucket name used to store configuration files.\n" +
                         "sleeper.config.bucket: sleeper-test-instance-config\n")
                 // Then check properties in sequence to check spacing between them
-                .contains("# The amount of memory (GB) the athena composite handler has\n" +
-                        "sleeper.athena.handler.memory: 4096\n" +
+                .contains("# The loging level for Parquet logs.\n" +
+                        "sleeper.logging.parquet.level: null\n" +
                         "\n" +
-                        "# The timeout in seconds for the athena composite handler\n" +
-                        "sleeper.athena.handler.timeout.seconds: 900\n" +
+                        "# The loging level for AWS logs.\n" +
+                        "sleeper.logging.aws.level: null\n" +
                         "\n" +
-                        "# The number of days before objects in the spill bucket are deleted.\n" +
-                        "sleeper.athena.spill.bucket.ageoff.days: 1")
+                        "# The loging level for everything else.\n" +
+                        "sleeper.logging.root.level: null")
                 // Then check property with multi-line description
                 .contains("# The minimum number of files to read in a compaction job. Note that the state store\n" +
                         "# must support atomic updates for this many files. For the DynamoDBStateStore this\n" +
                         "# is 11. It can be overridden on a per-table basis.\n" +
                         "# (NB This does not apply to splitting jobs which will run even if there is only 1 file.)\n" +
                         "# This is a default value and will be used if not specified in the table.properties file\n" +
-                        "sleeper.default.compaction.files.batch.size: 11");
+                        "sleeper.default.compaction.files.batch.size: 11")
+                // Then check property groups are printed
+                .contains(PropertyGroup.all().stream()
+                        .map(PropertyGroup::getDescription)
+                        .toArray(String[]::new));
```

Comment:
We need to assert the value here. This duplicates the implementation. We can just check a couple of groups, but assert what it should actually look like. It's not clear to me whether it'll read well.